### PR TITLE
refactor: Switch to localkms in createkeyset wasm

### LINKED ISSUE #1710: Switch to localkms in createkeyset wasm

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1710

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/client/didexchange/client_test.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func TestClient_CreateInvitation(t *testing.T) {
 				didexchange.DIDExchange: svc,
 				route.Coordination:      &mockroute.MockRouteSvc{},
 			},
-			KMSValue:             &mockkms.CloseableKMS{CreateEncryptionKeyValue: "sample-key"},
+			LegacyKMSValue:       &mockkms.CloseableKMS{CreateEncryptionKeyValue: "sample-key"},
```
Comment (troyronda) [inline pkg/client/didexchange/client_test.go:143]: What's the `LegacyKMS` change?

Comment (fqutishat) [reply pkg/client/didexchange/client_test.go:143]: it's bug in unit-test. put legacykms value in wrong field

File: pkg/controller/command/kms/command.go [hunk 2]
```diff
@@ -58,15 +77,34 @@ func (o *Command) GetHandlers() []command.Handler {
...
+	pubKeyBytes, err := o.exportPubKeyBytes(keyID)
 	if err != nil {
 		logutil.LogError(logger, commandName, createKeySetCommandMethod, err.Error())
 		return command.NewExecuteError(CreateKeySetError, err)
 	}
 
 	command.WriteNillableResponse(rw, &CreateKeySetResponse{
-		EncryptionPublicKey: encryptionPublicKey,
-		SignaturePublicKey:  signaturePublicKey,
+		KeyID:     base64.RawURLEncoding.EncodeToString([]byte(keyID)),
```
Comment (rolsonquadras) [inline pkg/controller/command/kms/command.go:106]: do we need base64 encoding for keyID ?

Comment (fqutishat) [reply pkg/controller/command/kms/command.go:106]: yes

**Pull request merged**